packages/contracts/test/voting/token-voting.ts

```diff
@@ -167,52 +201,69 @@ describe('TokenVoting', function () {
     it('reverts if the user is not allowed to create a proposal and minProposerPower > 1 is selected', async () => {
       votingSettings.minProposerVotingPower = 123;
 
+      await setBalances([
+        {
+          receiver: signers[1].address,
+          amount: votingSettings.minProposerVotingPower,
```

Comment:
set the amount to `votingSettings.minProposerVotingPower - 1` and safe the call to `setTotalSupply`

Reply:
Clarified the test in e78f9088d9efe40e038e3f1dc7e6a61ef03c1d53